src/gui/widgets/Knob.cpp

```diff
@@ -101,6 +101,31 @@ void Knob::initUi( const QString & _name )
 	setInnerRadius( 1.0f );
 	setOuterRadius( 10.0f );
 	setFocusPolicy( Qt::ClickFocus );
+
+	// This is a workaround to enable style sheets for knobs which are not styled knobs.
+	//
+	// It works as follows: the palette colors that are assigned as the line color previously
+	// had been hard coded in the drawKnob method for the different knob types. Now the
+	// drawKnob method uses the line color to draw the lines. By assigning the palette colors
+	// as the line colors here the knob lines will be drawn in this color unless the stylesheet
+	// overrides that color.
+	switch (knobNum())
+	{
+	case knobSmall_17:
+	case knobBright_26:
+	case knobDark_28:
+		setlineColor(QApplication::palette().color( QPalette::Active, QPalette::WindowText ));
+		break;
+	case knobGreen_17:
+		setlineColor(QApplication::palette().color( QPalette::Active, QPalette::BrightText ));
+		break;
+	case knobVintage_32:
+		setlineColor(QApplication::palette().color( QPalette::Active, QPalette::Shadow ));
+		break;
+	default:
+		break;
+	}
+
```

Comment:
Very interesting.... so `lineColor` was already part of the API but wasn't used for this particular draw function?  Elegant.